OSSSpeechKit/Classes/OSSSpeech.swift

```diff
@@ -165,6 +167,9 @@ public class OSSSpeech: NSObject {
 
     // MARK: - Private Properties
 
+    private var audioRecorder:AVAudioRecorder?
+    private var audioFileURL:URL!
```

Comment:
```suggestion
    private var audioFileURL: URL?
```

Reply:
Could you please add a code comment about why we need this? Also, I have made it an optional since we don't know if it will ever get created or used, so if we attempt to access it, it might cause a crash if it has no value.